pkg/virt-handler/virtwrap/api/converter.go

```diff
@@ -325,6 +325,18 @@ func Convert_v1_VirtualMachine_To_api_Domain(vm *v1.VirtualMachine, domain *Doma
 		}
 	}
 
+	if vm.Spec.Domain.CPU != nil {
+		domain.Spec.CPU.Topology = &CPUTopology{
+			Sockets: 1,
+			Cores:   vm.Spec.Domain.CPU.Cores,
+			Threads: 1,
+		}
+		domain.Spec.VCPU = &VCPU{
+			Placement: "static",
```

Comment:
So, with static placement without specifying the cpuset, we're just pinning the qemu process to all visible cores.

This seems fine. In the future we're planning on cpu pinning and numa to be something handled by the container runtime... right?  If that's the case, I think our work's done here for the domain xml part.

Reply:
Yes, we should work on being able to inherit cpu pinning featrues of the container runtime.

Reply:
Yes, until we have something in k8s which allows pinning, we are done. However once we have that, we will still have to do the pinning for individual cores inside the assigned cpuset.

Reply:
@nertpinx @ehabkost - Just to put this onto your radar. This will be something interesting to solve.